x-pack/plugins/cases/public/components/create/custom_fields.test.tsx

```diff
@@ -101,6 +103,7 @@ describe('CustomFields', () => {
           customFields: {
             [textField.key]: 'hello',
             [toggleField.key]: true,
+            [customFieldsConfigurationMock[3].key]: false,
```

Comment:
I think because `customFieldsConfigurationMock[2]` is type text, is not required and has no default value.

This means on render there will be no value set so the validation for that field fails.

(See `x-pack/plugins/cases/public/components/custom_fields/text/create.tsx` and `x-pack/plugins/cases/public/components/custom_fields/text/config.ts`.)